### DISCUSSION TITLE
Semi-colon by itself on a line after the SQL == bad?

### DISCUSSION THREAD

**Discussion by jsteinfort-amd** [dbeaver/dbeaver#16261] [Q&A] [answered]
I recently upgraded 22.0.3 and now it seems a semicolon that ends a SQL statement cannot be placed by itself on a line, after the SQL.

I have too many scripts that have the semicolon placed like this.

Is this behavior configurable?  For some reason, I'm unable to spot this change in the release notes.  

Any help is much appreciated.  For now, I have reverted to my last used version: 22.0.1.

**Comment by Matvey16** [comment, marked as answer]
This will be fixed in the next version https://github.com/dbeaver/dbeaver/issues/16212
For now there is a workaround mentioned [here](https://github.com/dbeaver/dbeaver/issues/16077#issuecomment-1099991921)
